code/ai/aibig.cpp

```diff
@@ -691,7 +690,7 @@ void ai_big_chase_attack(ai_info *aip, ship_info *sip, vec3d *enemy_pos, float d
 		aip->prev_goal_point = En_objp->pos;
 	} else {
 		//	If moving slowly, maybe evade incoming fire.
-		if (Pl_objp->phys_info.speed < 3.0f) {
+		if (Pl_objp->phys_info.speed < The_mission.ai_profile->standard_strafe_when_below_speed) {
```

Comment:
This block seems to be controlling more than just strafe behavior, might it be better to leave this `3.0` here, and instead use the strafe below speed lower down when it's actually considering whether to enter `AIM_STRAFE`?

Reply:
Thanks, I've pulled that section out